### PR TITLE
Hide multiple choice options

### PR DESCRIPTION
**Pull request by bjornrl**
Works on my local <3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,13 +108,12 @@
...
     "react-i18next": "11.11.0",
     "react-icons": "^4.3.1",
-    "react-markdown": "^8.0.0",
+    "react-markdown": "^7.0.1",
     "react-monaco-editor": "^0.43.0",
     "react-redux": "^7.2.4",
     "react-refresh": "^0.8.3",
     "react-router-dom": "^5.3.0",
     "react-syntax-highlighter": "^15.4.4",
-    "remark-gfm": "^3.0.1",
```
Comment (oriooctopus) [inline package.json]: why is this removed?

File: package.json [hunk 2]
```diff
@@ -123,6 +122,7 @@
     "sass-loader": "^10.0.5",
     "sass.js": "^0.11.1",
     "semver": "7.3.2",
+    "strapi-plugin-import-export-content": "^0.4.1",
```
Comment (oriooctopus) [inline package.json]: Why was this added?

**Review by oriooctopus — CHANGES_REQUESTED** [review]
Before I review more, please look through your PR and review it yourself. Also include a video or screenshots of the feature

File: src/components/Challenges/Challenge.query.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import gql from 'graphql-tag';
```
Comment (oriooctopus) [inline src/components/Challenges/Challenge.query.tsx]: This file was replaced with Step.query.tsx a few commits ago. I think you probably made a mistake while rebasing. It needs to be removed. If you made any changes here, you probably meant for them to be in Step.query.tsx

File: src/components/Challenges/MultipleChoiceChallenge/MultipleChoiceChallenge.tsx [hunk 4]
```diff
@@ -72,20 +80,31 @@ export const MultipleChoiceChallenge = ({
...
+      {showOptions &&
+        options
+          ?.filter(removeEmpty)
+          .map(({ incorrectChoiceExplanation, text }, index) => (
+            <MultipleChoiceChallengeOption
+              text={text}
+              incorrectChoiceExplanation={incorrectChoiceExplanation}
+              isOptionSelected={isOptionSelected(index)}
+              onClickOption={() => onClickOption(index)}
+              showOptionIncorrectExplanation={shouldShowOptionIncorrectExplanation(
```
Comment (oriooctopus) [inline src/components/Challenges/MultipleChoiceChallenge/MultipleChoiceChallenge.tsx]: a ternary would be cleaner here than two && statements

**Review by oriooctopus — CHANGES_REQUESTED** [review]

**Comment by oriooctopus** [comment]
Overall looks good, just needs a few changes

File: package.json [hunk 5]
```diff
@@ -108,13 +108,12 @@
     "react-dom": "^17.0.0",
     "react-i18next": "11.11.0",
     "react-icons": "^4.3.1",
-    "react-markdown": "^8.0.0",
```
Comment (andypants888) [inline package.json]: Why are going down a version 8 --> 7? Is there a compatibility issue?

**Comment by bjornrl** [comment]
https://user-images.githubusercontent.com/87340318/159588484-bcdd4c5f-ce90-4ac9-9f19-a3f6e87b66d4.mov

Screen recording of feature

**Comment by andypants888** [comment]
I'm not too familiar with this section of the code, so I don't have much feedback. Although you can bet I'll ask you about it when I eventually work it! @bjornrl

File: src/components/Challenges/MultipleChoiceChallenge/MultipleChoiceChallenge.tsx (hunk 4 above)
Comment (bjornrl) [reply src/components/Challenges/MultipleChoiceChallenge/MultipleChoiceChallenge.tsx]: Fixed!

File: package.json (hunk 2 above)
Comment (bjornrl) [reply package.json]: Dont know, i never installed it...

**Comment by oriooctopus** [comment]
make sure to add the photo/video to the description next time, not the comments

File: package.json (hunk 5 above)
Comment (oriooctopus) [reply package.json]: Yes this is a good catch. Why are we going down a version?

**Review by oriooctopus — APPROVED** [review]
